examples/signal-entry/src/lib.rs

```diff
@@ -18,15 +18,16 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
-use gstd::errors::SignalCode;
+// We can't depend on gstd because it declares panic handler, so we just use gcore.
```

Comment:
you can still depend on gstd wout panic-handler feature, but it will cause tests compiling cargo issue